src/components/reviews/CreateReview.jsx

```diff
@@ -9,7 +11,9 @@ class CreateReview extends React.Component {
       recommended: null,
       characteristics: {}
```

Comment:
Curious- do you not have to initialize all properties of state that you plan to use? If you setstate with a property that doesn't exist, does it simply get added? I just realized I have always initialized state with some sort of default value, but maybe that isn't necessary.

Reply:
You don't have to for the sake of the state itself, but if you're rendering something using state you have to initialize it. Since I'm not initializing, I don't have to initialize anything except the rating and the recommended properties. I also have to initialize the characteristics container so I don't get referenceError when I try to add something to it.